packer/windows.pkr.hcl

```diff
@@ -10,6 +10,12 @@ variable "docker_repo" {
   default = ""
 }
 
+variable "execute_command" {
+  type = string
+  # execute command for vm instances, containers will overwrite this
+  default = "powershell -executionpolicy bypass \"& { if (Test-Path variable:global:ProgressPreference){$ProgressPreference='SilentlyContinue'}; $ErrorActionPreference = 'Stop' ;. {{.Vars}}; &'{{.Path}}'; exit $LastExitCode }\""
```

Comment:
Explain why we are doing this, and particularly why we are overwriting it for docker containers.